ci_framework/roles/operator_build/tasks/build.yml

```diff
@@ -0,0 +1,147 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Set default api path for {{ operator.name }}
+  ansible.builtin.set_fact:
+    operator_api_path: "github.com/{{ cifmw_operator_build_org }}/{{ operator.name }}/api"
+
+- name: Update the go.mod file in meta operator for provided PR_SHA
+  ansible.builtin.shell: |
+    go mod edit -replace {{ operator_api_path }}=github.com/{{ operator.pr_owner }}/api@{{ operator.pr_sha }}
+    go mod tidy
+  args:
+    chdir: "{{ cifmw_operator_build_meta_src }}"
+  when:
+    - cifmw_operator_build_meta_build
+    - operator.name != cifmw_operator_build_meta_name
+    - operator.pr_owner is defined
+    - operator.pr_sha is defined
+
```

Comment:
Based on https://github.com/openshift/release/blob/master/ci-operator/step-registry/openstack-k8s-operators/build/openstack-k8s-operators-build-commands.sh
```
go mod edit -replace github.com/${ORG}/${BASE_OP}/api=github.com/${REPO_NAME}/api@${API_SHA}
  go mod tidy
  pushd ./apis/
  go mod edit -replace github.com/${ORG}/${BASE_OP}/api=github.com/${REPO_NAME}/api@${API_SHA}
  go mod tidy
  popd
```
We already have added tasks for {{ operator.name }}/api. We also need add the same task for  {{ operator.name }}/apis/go.mod file.

Some of the operators does not have *apis* directory, we need to make sure first it exists and then apply the above task.

Rest looks good.

Reply:
is that one callable via some `make` target?